### PR TITLE
add column that combine months and years in a single report

### PR DESCRIPTION
**Pull request by czue**
this is a bit of a work in progress, but i'd like to make some reports off it.

@nickpell

### CODE REVIEW

**Comment by millerdev** [comment]
would it make sense to add tests?

**Comment by czue** [comment]
yeah, i'd like to. will try and clean this up a bit before tomorrow's deploy, but will push to just get it in if I don't get a chance to

**Comment by millerdev** [comment]
The test failure looks real
```
======================================================================
ERROR: test_format_both (corehq.apps.userreports.tests.test_columns.TestPercentageColumn)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/tests/test_columns.py", line 287, in test_format_both
    self.assertEqual('33% (1/3)', wrapped.format_fn()({'num': 1, 'denom': 3}))
AttributeError: 'PercentageColumn' object has no attribute 'format_fn'
```

File: corehq/apps/userreports/tests/test_columns.py [hunk 1]
```diff
@@ -301,3 +301,4 @@ def _test_spec(self):
                 "type": "field",
             }
         }
+n
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_columns.py]: F821 undefined name 'n'

**Comment by czue** [comment]
thanks, should be fixed

**Comment by czue** [comment]
whoops one sec

**Comment by czue** [comment]
:+1:

**Comment by czue** [comment]
@millerdev added tests in https://github.com/dimagi/commcare-hq/pull/6125

**Pull request merged**